fhd_core/calibration/vis_calibrate.pro

```diff
@@ -168,7 +168,7 @@ FOR iter=0,calibration_flag_iterate DO BEGIN
     t2+=t3_a-t2_a
     
     IF Keyword_Set(flag_calibration) THEN vis_calibration_flag,obs,cal,n_tile_cut=n_tile_cut,_Extra=extra
-    IF n_tile_cut EQ 0 THEN BREAK
+
```

Comment:
Right, but it should just break out of the FOR loop (not crash), which is exactly what you want in that case. This loop should only run a single time in any case, unless calibration_flag_iterate has been set. It looks like what it really needs is n_tile_cut=0 set just before the loop, so that it is defined. Or, change it to IF Keyword_Set(n_tile_cut) THEN BREAK